app/views/shared/_topic_page_breadcrumbs.html.erb

```diff
@@ -0,0 +1,18 @@
+<%
+  column_two_thirds ||= false
+
+  column_classes = %w()
```

Comment:
This array should [use square brackets][1].

```suggestion
  column_classes = %w[]
```

[1]: https://rubystyle.guide/#percent-literal-braces

Reply:
Fixed in https://github.com/alphagov/collections/pull/2781